packages/starknet-snap/src/estimateFee.ts

```diff
@@ -98,7 +99,10 @@ export async function estimateFee(params: ApiParams) {
     if (accountDeployed) {
       // This condition branch will be removed later when starknet.js
       // supports estimateFeeBulk in rpc mode
-      estimateFeeResp = await estimateFeeUtil(network, senderAddress, senderPrivateKey, txnInvocation);
+      // We should check here what is the CairoVersion to get the correct fees calculation
```

Comment:
please rollback to estimateFeeResp = await estimateFeeUtil(network, senderAddress, senderPrivateKey, txnInvocation);